src/test/java/ru/training/at/hw3/page/DifferentElementsPage.java

```diff
@@ -0,0 +1,69 @@
+package ru.training.at.hw3.page;
+
+import ru.training.at.hw3.utils.WaitActions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class DifferentElementsPage {
+
+    private WaitActions waitActions;
+    private WebDriver webDriver;
```

Comment:
I suppose that these fields could be extracted to the **Base class**

Reply:
I used to have a BasePage class but I thought it was too small to keep it. Now restored